### ISSUE TITLE
Prometheus Block Storage Minimum Size (on vultr.com)

### ISSUE THREAD

**Issue by darkpixel** [kontena/lens#5398]
I think this is similar to #3587.
When deploying the bundled Prometheus stack on vultr.com, it fails to create the volume with an error:

```failed to provision volume with StorageClass "vultr-block-storage-hdd": rpc error: code = Internal desc = {"error":"Unable to add block storage: Failed to add block storage device. Invalid size (20GB) - Must be between 40GB and 40000GB.","status":400}```

Vultr doesn't allow volumes smaller than 40 GB.  Maybe the volume size could be customizable before deployment?

I can't directly edit the PVC to adjust it to 40 GB:
```PersistentVolumeClaim "data-prometheus-0" is invalid: spec: Forbidden: spec is immutable after creation except resources.requests for bound claims```

**Comment by Nokel81** [comment]
Can you deploy a helm Prometheus stack instead? This won't be a very high priority for us.

**Comment by darkpixel** [comment]
Understood.  I just figured I would bring it to your attention.  Thanks!

**Comment by Nokel81** [comment]
Yes thank you. Though having such a large minimum size is very strange.

**Comment by lopnu** [comment]
Same here, plz help, I don't need the helm Prometheus stack, how to deploy minimal required lens-metrics?

**Comment by lopnu** [comment]
I found a simple solution:
1. I copied data-prometheus-0 claim yaml
2. Removed the claim with 20GB
3. Applied the yaml from step 1 with changed size to 40GB
4. Turned on Prometheus in Lens config
